atom/browser/net/atom_url_request.h

```diff
@@ -43,6 +43,7 @@ class AtomURLRequest : public base::RefCountedThreadSafe<AtomURLRequest>,
   void PassLoginInformation(const base::string16& username,
                             const base::string16& password) const;
   void SetLoadFlags(int flags) const;
+  void GetUploadProgress(mate::Dictionary& progress);
```

Comment:
`URLRequest::GetUploadProgress()` is const, so I think the compiler should be happy with making this new method const